src/styles/isomer-template/homepage/announcements.scss

```diff
@@ -0,0 +1,37 @@
+.announcements-divider {
+  height: 1px;
+  color: #f9f9f9;
+}
+
+.announcements-announcement-title {
+  font-size: 1.625rem;
+  font-weight: 700;
+  line-height: 2rem;
+}
+
+.announcements-announcement-subtitle {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.375rem;
+}
+
+.announcements-announcement-link {
+  text-transform: unset !important;
```

Comment:
May I get more context to this? why do we need to `!important` here ah? I was under the impression this is what we would be actively moving away from, is this assumption wrong?

Reply:
This will probably change! I originally did it this way to get access to the site-specific secondary colour + hover (could only access it using `.bp-button`, which had the `text-transform` set). Will adjust the template + this preview once the other changes are completed.

Reply:
Adjusted in bb34fae!